### PR TITLE
benchmarks: make latency benchmark less noisy

### LINKED ISSUE #2566: max latency benchmark is too noisy
Screen Shot 2019-06-22 at 12 13 23 PM

Currently we are plotting the absolute max latency - but we could get a smoother, more meaningful plot by using the 90% slowest requests by using `wrk --latency`

### PR DESCRIPTION
**Pull request by bartlomieju**
Closes #2566 

With this change we're tracking 90th percentile of latency.

I did not rename stat field (it's still `max_latency`) but if you want it renamed @ry let me know.

### CODE REVIEW

File: tools/util.py [hunk 1]
```diff
@@ -384,7 +384,7 @@ def parse_wrk_output(output):
                                                   line)
         if stats['max_latency'] is None:
             stats['max_latency'] = extract_max_latency_in_milliseconds(
-                r'Latency(?:\s+(\d+.\d+)([a-z]+)){3}', line)
+                r'\s+90%(?:\s+(\d+.\d+)([a-z]+))', line)
```
Comment (ry) [inline tools/util.py]: I think it would be better to use the 99% (sorry that I said 90% in the issue)

Comment (bartlomieju) [reply tools/util.py]: No prob, updated :+1:

**Comment by ry** [comment]
Traceback (most recent call last):
  File "/Users/travis/build/denoland/deno/tools/util_test.py", line 52, in test_parse_wrk_output
    assert stats['max_latency'] == 2.43
AssertionError

**Comment by bartlomieju** [comment]
@ry sorry too fast 😅 it should be green now

**Pull request merged**